etc/inc/interfaces.inc

```diff
@@ -3459,10 +3459,16 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 		$dhcp6cconf .= "	script \"{$g['varetc_path']}/dhcp6c_{$interface}_script.sh\"; # we'd like some nameservers please\n";
 		$dhcp6cconf .= "};\n";
 	} else {
+		$iflist = link_interface_to_track6($interface);
+		foreach ($iflist as $ifname => $ifcfg) {
+			if (is_numeric($ifcfg['track6-prefix-id']))
```

Comment:
Just to be sure, check if $ifcfg['track6-prefix-id'] is set before call is_numeric()